src/App.vue

```diff
@@ -1,17 +1,194 @@
+      while(ctr > 0) {
+          index = Math.floor(Math.random() * ctr)
+          ctr --,
+          array = [arr1[index], arr1[ctr]] = [arr1[ctr], arr1[index]]
+      }
+      return array
+    },
+    checkAnswer() {
+      if (this.guessValue === this.RhymusGame.currentCard.answer) {
+          this.isCorrect = true
+          this.totalCorrect++
+          this.correctAnswer()
+          } else if (this.guessValue === '' || undefined) {
+          alert('Please enter a guess!')
+          } else {
+          this.isCorrect = false
+          this.totalWrong++
+          this.incorrectAnswer()
+      }
+    },
+    guessValueUpdate(val) {
+      this.guessValue = val
+    },
+    correctAnswer() {
+      this.classNameObject.cardBlock.correct = this.isCorrect ? true : null
+      this.classNameObject.guessElement.correct = this.isCorrect ? true : null
+      this.cardNumber++
+      this.correctGuesses++
+      setTimeout(() => {
+          this.classNameObject.cardBlock.correct = false
+          this.classNameObject.guessElement.correct = false
+          if (this.cardNumber < this.puzzlesArray.length) {
+              this.loadCard(this.puzzlesArray[this.cardNumber])
+          }
+          else {
+              this.gameOver()
+          }
+      }, 800)
+    },
+    incorrectAnswer() {
+      this.classNameObject.cardBlock.incorrect = this.isCorrect ? null : true
+      this.classNameObject.guessElement.incorrect = this.isCorrect ? null : true
+      this.incorrectGuesses++
+      this.guessValue = ''
+
+      setTimeout(() => {
+          this.classNameObject.cardBlock.incorrect = false
+          this.classNameObject.guessElement.incorrect = false
+      }, 500)
+      this.wrongText = `incorrect guesses: ${this.incorrectGuesses}`
+      //if there are more than 4 incorrect guesses on a single card the player loses
+      if (this.incorrectGuesses > 1 && this.incorrectGuesses < 4) {
+          this.stylingObject.hint.display = 'flex'
+          this.hintText = 'Hint: ' + this.capitalizeFirstLetter
+      } else {
+          this.gameOver()
+      }
+    },
+    gameOver() {
+        // Accounts for array starting at 0
+      if (this.timeRemaining === 0 || this.correctGuesses + 1 === this.puzzlesArray.length || this.incorrectGuesses >= 4) {
+          this.isFirstGame = (this.isFirstGame) ? false : this.isFirstGame
+          this.isGameStarted = false
+          this.classNameObject.timerDisplay.running = false
+          this.classNameObject.timerDisplay.gameover = true
+          this.RhymusGame.rhymeElementText = 'Game Over!'
+          this.stylingObject.timerDisplay.background = 'var(--danger)'
+          this.stylingObject.cardBlock.background = 'var(--danger)'
+          this.stylingObject.hint.display = 'none'
+      }
+    }
+  },
+  computed: {
+    capitalizeFirstLetter() {
+      return this.RhymusGame.currentCard.hint.charAt(0).toUpperCase() + this.RhymusGame.currentCard.hint.slice(1).toLowerCase()
+    }
```

Comment:
Good use of `computed` to make modifications of data prior to its application without modifying the source. Avoiding mutations is great practice.